### PR TITLE
Issue 1098 rp name logo

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@warner, @lloyd - a new pull request for the RP name/logo using what we learned about input parameters.  Would you guys mind reviewing this?

### CODE REVIEW

**Comment by warner** [comment]
Looks good to me.. @lloyd, wanna cross-check?

**Comment by lloyd** [comment]
cross checking...

**Comment by lloyd** [comment]
am confused about card.png being checked in.  It's browserid branded?  What is the purpose of this file and where does is show up?

File: resources/static/dialog/controllers/dialog.js [hunk 1]
```diff
@@ -168,6 +173,17 @@ BrowserID.Modules.Dialog = (function() {
           params.privacyURL = fixupURL(origin_url, paramsFromRP.privacyPolicy);
         }
 
+        if (paramsFromRP.logoURL) {
+          // Until we have our head around the dangers of data uris and images
+          // that come from other domains, only allow absolute paths from the
+          // origin.
+          params.logoURL = fixupAbsolutePath(origin_url, paramsFromRP.logoURL);
```
Comment (lloyd) [inline resources/static/dialog/controllers/dialog.js]: I thought we agreed this was going to be called siteLogo: https://groups.google.com/group/mozilla.dev.identity/browse_thread/thread/d4b1de64ce6cc6e5/3c4fa265a6ab1c9b?lnk=gst&q=siteName#3c4fa265a6ab1c9b

File: resources/static/dialog/controllers/dialog.js [hunk 2]
```diff
@@ -168,6 +173,17 @@ BrowserID.Modules.Dialog = (function() {
...
+        }
+
+        if (paramsFromRP.name) {
+          params.name = _.escape(paramsFromRP.name);
```
Comment (lloyd) [inline resources/static/dialog/controllers/dialog.js]: likewise, I thought this parameter was going to be called siteName

**Comment by lloyd** [comment]
three issues mentioned above, I'll commit a change to address two of them (api property naming).

**Comment by lloyd** [comment]
rebased against current dev, reviewed code.  The only thing blocking this PR from merge IMO is stomlinson weighing in on the branding in `card.png`, as I think at this point in dev, BrowserID never should be displayed to the user, only persona.

Once that's figured out, I'm r+ to merge to dev.

**Comment by lloyd** [comment]
r+

**Pull request merged**